lib/npa.js

```diff
@@ -87,6 +88,22 @@ function resolve (name, spec, where, arg) {
   }
 }
 
+function toPurl (arg, reg) {
```

Comment:
(i assume there's a better place to get this defaultRegistry value from)

this sets the function's length correctly
```suggestion
const defaultRegistry = 'https://registry.npmjs.com'

function toPurl (arg, reg = defaultRegistry) {
```